runtime/README.md

```diff
@@ -227,6 +229,57 @@ Table: `INVOKE_DPE_COMMAND` output arguments
 | chksum      | u32           | Checksum over other output arguments, computed by Caliptra. Little endian.
 | data        | u8[...]       | DPE response structure as defined in the DPE iRoT profile.
 
+### QUOTE\_PCRS
+
+Generate a signed quote over all Caliptra hardware PCRs using the Caliptra PCR quoting key.
+All PCR values are hashed together for the quote
+
+Command Code: `0x5043_5251` ("PCRQ")
+
+`QUOTE_PCRs` takes no input arguments.
+
+Table: `QUOTE_PCRS` output arguments
+
+| **Name**     | **Type**   | **Description**
+| --------     | --------   | ---------------
+| chksum       | u32        | Checksum over other output arguments, computed by Caliptra. Little endian.
+| digest       | u8[48]     | Digest of all PCRs
+| reset\_ctrs  | u32[24]    | Reset counters for all PCRs
+| signature\_r | u8[48]     | R portion of the signature over the PCR quote.
+| signature\_s | u8[48]     | S portion of the signature over the PCR quote.
+
+### EXTEND\_PCR
+
+Extend a Caliptra hardware PCR
+
+Command Code: `0x5043_5245` ("PCRE")
+
+Table: `EXTEND_PCR` input arguments
+
+| **Name**     | **Type**      | **Description**
+| --------     | --------      | ---------------
+| chksum       | u32           | Checksum over other input arguments, computed by the caller. Little endian.
+| index        | u32           | Index of the PCR to extend
+| value        | u8[..]        | Value to extend into the PCR at `index`
+
+
+`EXTEND_PCR` returns no output arguments.
+
+### INCREMENT\_PCR\_RESET\_COUNTER
```

Comment:
Does this mean that DPE cannot attest to journey TCIs as a function of resets, i.e., journey TCIs are only a function of initial boot + hitless updates?

Reply:
It would say it is left up to the caller how to handle resets. The caller may choose to

* Treat a reset as a hitless update (and optionally record the reset in a log somewhere)
* Not remeasure (leave the existing pre-reset measurements) if code doesn't change
* Treat a reset as a cold reset for that component and remove the tree

I think the right behavior depends on the context and DPE should be capable of accommodating all of the above scenarios.

Reply:
Got it, but ultimately DPE is unaware of reset counters, it's just tracked on a per-PCR basis.

Reply:
Correct